### PR TITLE
Run storage command test in parallel

### PR DESCRIPTION
**Pull request by amitkrout**
## Was the change discussed in an issue?
fixes - part of #1473

## How to test changes?

make test-cmd-storage

### CODE REVIEW

**Comment by amitkrout** [comment]
ping @kadel @mik-dass @surajnarwade

File: tests/integration/cmd_storage_test.go [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+		})
+	})
+
+	Context("when running storage command without required flag(s)", func() {
+		It("should fail", func() {
+			helper.CopyExample(filepath.Join("source", "dotnet"), context)
+			helper.CmdShouldPass("odo", "component", "create", "dotnet", "dotnet", "--app", "dotnetapp", "--project", project, "--context", context)
+			stdErr := helper.CmdShouldFail("odo", "storage", "create", "pv1")
+			Expect(stdErr).To(ContainSubstring("Required flag"))
+			//helper.CmdShouldFail("odo", "storage", "create", "pv1", "-o", "json")
```
Comment (amitkrout) [inline tests/integration/cmd_storage_test.go]: @surajnarwade @mik-dass Can you please confirm if command fails then does it capture the output in json format ?

**Comment by amitkrout** [comment]
```
✗  Failed to create component with name testcmp. Please use `odo config view` to view settings used to create component. Error: build testcmp-testing-1 status Failed
[odo] github.com/openshift/odo/pkg/occlient.(*Client).WaitForBuildToFinish
[odo] 	/go/src/github.com/openshift/odo/pkg/occlient/occlient.go:1781
[odo] github.com/openshift/odo/pkg/component.Build
[odo] 	/go/src/github.com/openshift/odo/pkg/component/component.go:811
[odo] github.com/openshift/odo/pkg/component.CreateFromGit
[odo] 	/go/src/github.com/openshift/odo/pkg/component/component.go:165
[odo] github.com/openshift/odo/pkg/component.CreateComponent
[odo] 	/go/src/github.com/openshift/odo/pkg/component/component.go:397
[odo] github.com/openshift/odo/pkg/odo/cli/component.(*PushOptions).createCmpIfNotExistsAndApplyCmpConfig
[odo] 	/go/src/github.com/openshift/odo/pkg/odo/cli/component/push.go:175
[odo] github.com/openshift/odo/pkg/odo/cli/component.(*PushOptions).Run
[odo] 	/go/src/github.com/openshift/odo/pkg/odo/cli/component/push.go:203
[odo] github.com/openshift/odo/pkg/odo/genericclioptions.GenericRun
[odo] 	/go/src/github.com/openshift/odo/pkg/odo/genericclioptions/runnable.go:17
[odo] github.com/openshift/odo/pkg/odo/cli/component.NewCmdPush.func1
[odo] 	/go/src/github.com/openshift/odo/pkg/odo/cli/component/push.go:293
[odo] github.com/openshift/odo/vendor/github.com/spf13/cobra.(*Command).execute
[odo] 	/go/src/github.com/openshift/odo/vendor/github.com/spf13/cobra/command.go:702
[odo] github.com/openshift/odo/vendor/github.com/spf13/cobra.(*Command).ExecuteC
[odo] 	/go/src/github.com/openshift/odo/vendor/github.com/spf13/cobra/command.go:783
[odo] github.com/openshift/odo/vendor/github.com/spf13/cobra.(*Command).Execute
[odo] 	/go/src/github.com/openshift/odo/vendor/github.com/spf13/cobra/command.go:736
[odo] main.main
[odo] 	/go/src/github.com/openshift/odo/cmd/odo/odo.go:65
[odo] runtime.main
[odo] 	/usr/local/go/src/runtime/proc.go:201
[odo] runtime.goexit
[odo] 	/usr/local/go/src/runtime/asm_amd64.s:1333
[odo] unable to build testcmp-testing-1, error: Cloning "https://github.com/appuio/example-php-sti-helloworld" ...
```
/test integration

**Comment by amitkrout** [comment]
OpenShift CI hitting the same issue https://github.com/openshift/odo/pull/1816#issuecomment-504423076 twice.

/test integration

**Comment by amitkrout** [comment]
> OpenShift CI hitting the same issue [#1816 (comment)](https://github.com/openshift/odo/pull/1816#issuecomment-504423076) twice and the failure is coming from generic_test.go file
> 
> /test integration

Seems it was a flake

File: tests/integration/cmd_storage_test.go (hunk 1 above)
Comment (surajnarwade) [reply tests/integration/cmd_storage_test.go]: we dont have control over this as of now :|

**Pull request merged**